_docs/concepts/auth.md

```diff
@@ -0,0 +1,154 @@
+<!----- Conversion time: 5.4 seconds.
+
+
+Using this Markdown file:
```

Comment:
Hi, this document is missing the standard include headers we have in all our files. Besides, it also has some html, and I am not sure how it will work with rest of kramdown. Could you please point us to a hosted version fo the doc?